src/components/Wizard/__stubs__/schema.ts

```diff
@@ -0,0 +1,21 @@
+export const schema = {
+	type: 'object',
+	properties: {
+		documentDescription: {
+			type: 'string',
+			'bmi-isNotEmpty': true,
```

Comment:
Yes it is. The way the required rule works in ajv is not as you'd expect in a typical form validation situation. 
This method of adding a custom keyword is recommended in various places in the community.
The prefix is suggested to avoid conflicts with core rules.